### DISCUSSION TITLE
Possible bug on the repository page

### DISCUSSION THREAD

**Discussion by kitswas** [community/community#61813] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/90329875/1da8744b-d4c5-493c-9c05-3e7c763d8fae)

I am the sole contributor to this repository(as of now).  
Then why does it say 2 contributors?

Repo link:  
https://github.com/kitswas/VirtualGamePad-PC

**Comment by moonfury-ops** [comment]
Thank you
